packages/api/cms-api/src/public-upload/public-uploads.service.ts

```diff
@@ -38,4 +38,24 @@ export class PublicUploadsService {
 
         return publicUpload;
     }
+
+    async getFileById(id: string): Promise<PublicUpload> {
+        const file = await this.publicUploadsRepository.findOne(id);
+        if (!file) {
+            throw new NotFoundException();
+        }
+        return file;
+    }
+
+    async getFileStreamById(id: string): Promise<NodeJS.ReadableStream> {
```

Comment:
why pass the id insted of the PublicUpload object?

Reply:
This function no longer exists.